lms/static/js/verify_student/views/make_payment_step_view.js

```diff
@@ -105,10 +105,20 @@ var edx = edx || {};
                 self._getProductText( templateContext.courseModeSlug, templateContext.upgrade )
             );
 
-            // create a button for each payment processor
-            _.each(processors.reverse(), function(processorName) {
-                $( 'div.payment-buttons' ).append( self._getPaymentButtonHtml(processorName) );
-            });
+            if (processors.length === 0) {
+                // No payment processors are enabled at the moment, so show an error message
+                this.errorModel.set({
+                    errorTitle: gettext("We're currently experiencing technical problems."),
+                    errorMsg: gettext('Try the transaction again in a few minutes.'),
```

Comment:
@peter-fogg @rlucioni Based on Renzo's feedback, we might change the text to this (since the technical problems aren't with us):

All payment options are currently unavailable.
Try the transaction again in a few minutes.

Thoughts?